apps/connect/release_notes.md

```diff
@@ -1,21 +1,11 @@
 # ftrack Connect release Notes
 
-## v3.0.0rc3
-2024-03-27
+## v24.4.0
```

Comment:
```suggestion
## v3.0.0
```

Connect is a module, not using BL versioning right?

Reply:
ahhhhhh, Thank you!!!! good spot!